packages/babel-helper-function-name/src/index.js

```diff
@@ -2,7 +2,8 @@ import getFunctionArity from "babel-helper-get-function-arity";
 import template from "babel-template";
 import * as t from "babel-types";
 
-const buildPropertyMethodAssignmentWrapper = template(`
+const buildPropertyMethodAssignmentWrapper = template(
+  `
```

Comment:
Looks like we have a bunch of these too. It's an interesting case since it follows the rules for normal normal expressions, but seems weird when you see it. @vjeux any thoughts on this one?